### PR TITLE
Nms.ft.39.40.41.42.43.44.45.46.47

### PR DESCRIPTION
**Pull request by nmsyogesh**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by nmsyogesh** [comment]
JIRA issue for NMS_FT_41 raised at https://applab.atlassian.net/browse/NMS-200

File: testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java [hunk 1]
```diff
@@ -782,5 +784,232 @@ public void verifyFT77()
...
+ 				1, // callDisconnectReason
+ 				null)); // content
+        String expectedJsonResponse = createFailureResponseJson("<operator: Not Present>");
+
+ 		assertTrue(SimpleHttpClient.execHttpRequest(httpPost,
+ 				HttpStatus.SC_BAD_REQUEST, expectedJsonResponse,
+ 				ADMIN_USERNAME, ADMIN_PASSWORD));
+    }
+
+    @Ignore
```
Comment (frankhuster) [inline testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java]: Include link to JIRA issue in a //todo comment

**Comment by nmsyogesh** [comment]
@frankhuster
I have added JIRA issue in TODO comment.

**Comment by frankhuster** [comment]
please merge

**Comment by srivastavaAbhi** [comment]
Tracked with branch NMS.FT.KILKARICONTROLLERIT.
So closing this request.

**Pull request closed without merging**